crates/sui-e2e-tests/tests/coin_deny_list_tests.rs

```diff
@@ -1,18 +1,29 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use move_core_types::ident_str;
```

Comment:
Were you not able to do this as a transactional test? I personally just would find it easier to see what exactly we are testing here

Reply:
I wanted to do a bunch of consistency checks on the objects generated from publish, so I needed to write Rust tests anyway, and ended up just doing everything there.
I can follow up with more transactional tests. It's doable.